src/main/java/com/limechain/rpc/pubsub/subscriberchannel/Subscriber.java

```diff
@@ -0,0 +1,12 @@
+package com.limechain.rpc.pubsub.subscriberchannel;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.springframework.web.socket.WebSocketSession;
+
+@Data
+@AllArgsConstructor
+public class Subscriber {
+    private final String subscriptionId;
+    private final WebSocketSession session;
+}
```

Comment:
Data contains `ReqArgCons` which is effectively `AllArgCons` when all the fields are final.